src/server/send/send.sendgrid.js

```diff
@@ -36,6 +36,14 @@ export const sendLinkBySMS = (to: string, link: string) => {
   return client.messages.create(plivoPhoneNumber, to, text)
 }
 
+export const sendMagicCodeBySMS = (to: string, code: string) => {
```

Comment:
this is in the wrong file. its sendgrid file, dont know why other sms message is here. its something old unused.

Reply:
we also use twilio not plivo.

Reply:
done